records.py

```diff
@@ -209,6 +209,18 @@ def __init__(self, db_url=None):
 
         # Enable hstore if it's available.
         self._enable_hstore()
+        self.open = True
```

Comment:
it's just a boolean to keep track of the state of the connection. It can maybe be used on \_\_repr\_\_